<?xml version="1.0" encoding="UTF-8"?>
<robot generated="20141128 14:32:25.797" generator="Robot 2.8.4 (Python 2.7.6 on linux2)">
<suite source="/home/onnana/Documents/workspace/RobotBank/test/robot" id="s1" name="Robot">
<suite source="/home/onnana/Documents/workspace/RobotBank/test/robot/tests" id="s1-s1" name="Tests">
<suite source="/home/onnana/Documents/workspace/RobotBank/test/robot/tests/bank.txt" id="s1-s1-s1" name="Bank">
<test id="s1-s1-s1-t1" name="Retrieve customer balance">
<kw type="kw" name="steps.Given I create the account 1111 with balance 50">
<doc></doc>
<arguments>
</arguments>
<kw type="kw" name="robotLibrary.Create Account">
<doc></doc>
<arguments>
<arg>${account_number}</arg>
<arg>${balance}</arg>
</arguments>
<status status="PASS" endtime="20141128 14:32:26.033" starttime="20141128 14:32:26.033"></status>
</kw>
<status status="PASS" endtime="20141128 14:32:26.033" starttime="20141128 14:32:26.032"></status>
</kw>
<kw type="kw" name="steps.When I retrieve the account 1111">
<doc></doc>
<arguments>
</arguments>
<kw type="kw" name="robotLibrary.Visit Homepage">
<doc></doc>
<arguments>
</arguments>
<msg timestamp="20141128 14:32:26.037" level="WARN">Exception on / [GET]</msg>
<msg timestamp="20141128 14:32:26.039" level="FAIL">AppError: Bad response: 500 INTERNAL SERVER ERROR (not 200 OK or 3xx redirect for http://localhost:5000/)
'&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN"&gt;\n&lt;title&gt;500 Internal Server Error&lt;/title&gt;\n&lt;h1&gt;Internal Server Error&lt;/h1&gt;\n&lt;p&gt;The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.&lt;/p&gt;\n'</msg>
<status status="FAIL" endtime="20141128 14:32:26.039" starttime="20141128 14:32:26.034"></status>
</kw>
<status status="FAIL" endtime="20141128 14:32:26.039" starttime="20141128 14:32:26.034"></status>
</kw>
<doc></doc>
<tags>
</tags>
<status status="FAIL" endtime="20141128 14:32:26.040" critical="yes" starttime="20141128 14:32:26.031">AppError: Bad response: 500 INTERNAL SERVER ERROR (not 200 OK or 3xx redirect for http://localhost:5000/)
'&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN"&gt;\n&lt;title&gt;500 Internal Server Error&lt;/title&gt;\n&lt;h1&gt;Internal Server Error&lt;/h1&gt;\n&lt;p&gt;The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.&lt;/p&gt;\n'</status>
</test>
<doc>Example Robot tests for
Bank application</doc>
<metadata>
</metadata>
<status status="FAIL" endtime="20141128 14:32:26.040" starttime="20141128 14:32:25.828"></status>
</suite>
<doc></doc>
<metadata>
</metadata>
<status status="FAIL" endtime="20141128 14:32:26.041" starttime="20141128 14:32:25.825"></status>
</suite>
<doc></doc>
<metadata>
</metadata>
<status status="FAIL" endtime="20141128 14:32:26.042" starttime="20141128 14:32:25.797"></status>
</suite>
<statistics>
<total>
<stat fail="1" pass="0">Critical Tests</stat>
<stat fail="1" pass="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="1" pass="0" id="s1" name="Robot">Robot</stat>
<stat fail="1" pass="0" id="s1-s1" name="Tests">Robot.Tests</stat>
<stat fail="1" pass="0" id="s1-s1-s1" name="Bank">Robot.Tests.Bank</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20141128 14:32:26.037" level="WARN">Exception on / [GET]</msg>
</errors>
</robot>
